src/bluefinClient.ts

```diff
@@ -939,6 +941,41 @@ export class BluefinClient {
     return response;
   };
 
+  /**
+   * @description
+   * Add or remove user's subaccount (One Click Trading)
+   * @param accountAddress address to whitelist as a subaccount
+   * @returns ResponseSchema
+   */
```

Comment:
The  comments need improvement here. The description is not accurate to what's happening in the function and the param comment is not correct either as it's a custom interface comprising the `accountAddress` and the `accountsToRemove`